basket/news/tests/test_tasks.py

```diff
@@ -771,3 +786,10 @@ def test_timestamps_nothin_cached(self, cache_mock, sfmc_mock):
             'FXA_ID': data['uid'],
             'NewEmailAddress': data['email'],
         })
+
+
+class GmttimeTests(TestCase):
+    def test_basetime_provided(self):
+        basetime = parse_date('2018-05-18T13:21:54.188Z')
+        formatted_time = gmttime(basetime)
+        self.assertEqual(formatted_time, 'Fri, 18 May 2018 21:31:54 GMT')
```

Comment:
I'm not quite sure why gmttime adds 10 minutes to the time, but that's what it does.